--- src/schema.rs.unpatched	2023-10-30 10:24:59
+++ src/schema.rs	2023-10-30 11:23:34
@@ -1,9 +1,19 @@
 // @generated automatically by Diesel CLI.
 
 diesel::table! {
     people (id) {
+        rowid -> BigInt,
         id -> Text,
         first_name -> Text,
         last_name -> Text,
     }
 }
+
+diesel::table! {
+    people_fts (rowid) {
+        rowid -> Integer,
+        id -> Text,
+        first_name -> Text,
+        last_name -> Text,
+    }
+}
